server/src/main/java/io/druid/client/cache/Cache.java

```diff
@@ -39,6 +40,8 @@
 
   public boolean isLocal();
 
+  public Collection<String> getNamespaces();
```

Comment:
This was a leftover change from some functionality that I took out yesterday because it was only half-functional but mostly broken. I have removed the changes and will be restoring Cache and its implementations to their original former glory